src/pages/hegel/guides/science-of-logic-introduction.mdx

```diff
@@ -0,0 +1,184 @@
+
+This non-metaphysical, or formal, logic must continually seek a content outside of itself in order to become something meaningful. Formal logic must deal with objects of the world if it is to be determine something substantial, such as how various natural things behave. But Hegel's claim is that logic _does_ have a content of its own; it does have something internally meaningful&mdash;at a minimum _it is_ the concrete unity that holds together all the fixed, pre-established structures, be they syllogisms, propositions or other categories. It is not the fault of logic, Hegel says, that it seems empty, but it is, rather, the way in which it is being understood which is empty (Hegel, 2010, 28/21.32). If one begins to inspect how these various logical forms are connected, one has already begun to leave behind logic as fixed patterns with the implicit understanding that an integral unity exists. From there, the step to metaphysics is a small one&mdash;insofar as it is understood that metaphysics, generally speaking, is the philosophical inquiry into the nature of reality, existence and universe&mdash;since it is already given to understand that logic is something forms part of reality.
+
+### On the Use of the Term Science
+
+The term `science` (German: _Wissenschaft_) is often used by Hegel throughout his writings and signifies a larger context than what is commonly understood by the use of the term. Typically, science is understood to entail inquiries strictly into the natural world or what can be proven empirically. Science thus is therefore knowledge of empirical reality. Hegel, along with Kant, Fichte and others of his contemporaries, expand the notion to include also what can be proven logically. While science, for Hegel, still concerns empirical knowledge, it is also knowledge of non-empirical items such as logic, metaphysics, ethics, aesthetics, religion and even knowledge itself. It is perhaps best to think of knowledge whenever Hegel speaks about science, insofar as knowledge is a provable conception (logically or empirically) of something that is neither exclusively objective or subjective. The class of knowledge that is particularly concerned with logic, metaphysics or knowledge as such is frequently connoted as pure knowledge or pure science; "pure" because it is knowledge about knowledge, or science about science, or the subject matter of logic is strictly not intermingled with anything other than logic itself.
```

Comment:
"While science, for Hegel, still concerns empirical knowledge" - off hand, Hegel does tend to refer to science of empirical objects as natural science or empirical physics.

Reply:
This is a good point, but I think we'll have to add it in later since §246 is in enc nature section and I don't have that on hand right now.